### ISSUE TITLE
On the distance between two neural networks and the stability of learning

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#47]
On the distance between two neural networks and the stability of learning
Jeremy Bernstein, Arash Vahdat, Yisong Yue, Ming-Yu Liu
This paper relates parameter distance to gradient breakdown for a broad class of nonlinear compositional functions. The analysis leads to a new distance function called deep relative trust and a descent lemma for neural networks. Since the resulting learning rule seems not to require learning rate grid search, it may unlock a simpler workflow for training deeper and more complex neural networks. Please find the Python code used in this paper here: this https URL.

![Screen Shot 2020-08-26 at 17 49 23](https://user-images.githubusercontent.com/8721858/91282619-85a4d580-e7c4-11ea-8a3b-1ea36e68441b.png)
![Screen Shot 2020-08-26 at 17 50 32](https://user-images.githubusercontent.com/8721858/91282890-e0d6c800-e7c4-11ea-8f71-cdabcc607c0c.png)
![Screen Shot 2020-08-26 at 17 51 08](https://user-images.githubusercontent.com/8721858/91282897-e3392200-e7c4-11ea-952d-5fc903469fad.png)
![Screen Shot 2020-08-26 at 17 52 11](https://user-images.githubusercontent.com/8721858/91282931-edf3b700-e7c4-11ea-9efb-d11f1b82652e.png)
